### ISSUE TITLE
Connection close automatically

### ISSUE THREAD

**Issue by Geocomt** [googlesamples/android-nearby#58]
I am using Nearby on 2 tablets with strategy 'P2P_STAR'.

The first connection work fine and keep sending message a long time, but after a while (6 - 7 hours) the devices disconnect automatically.

¿Has the serves token a expiration time? ¿Why device lost her connection automatically?

**Comment by brianduff** [comment]
I don't think we expect connections to last for this length of time in general. It's highly likely that the persistent process in Android, which manages these connections could be restarted within that timeframe for a variety of reasons, and we don't attempt to reestablish connections that are lost under these circumstances.

Can you give us some more info about what you're doing / the usecase you have - very long lived connections aren't something we've commonly seen; there might be an alternative approach we can suggest.
